heliopy/data/sunspot.py

```diff
@@ -0,0 +1,21 @@
+import requests  # To download file
+import datetime  # To find current date/time
+import pandas  # For Pandas Dataframe
+import os  # For extra stuff
+
+data_source = 'http://www.sidc.be/silso/INFO/sndtotcsv.php'  # File Link
+header_ = ['Y', 'M', 'D', 'DecD', 'Daily', 'Std Dev', 'No Obs', 'Def/Prov Ind']
+
+
+def data():

```

Comment:
Can you rename this to `daily()`. It also needs a docstring directly underneath, something like "Daily sunspot number import. See http://www.sidc.be/silso/infosndtot for more details"

Reply:
Oh yes, I'll add more functions to import different datasets as well, like monthly() and yearly()